src/fixit/api.py

```diff
@@ -23,7 +24,8 @@ def _make_result(path: Path, violations: Iterable[LintViolation]) -> Iterable[Re
     except Exception as error:
         # TODO: this is not the right place to catch errors
         logger.debug("Exception while linting", exc_info=error)
-        yield Result(path, violation=None, error=error)
+        tb = traceback.format_exc()
+        yield Result(path, violation=None, error=error, traceback=tb)
```

Comment:
maybe attach the traceback here and format at the place we're printing results?

Reply:
Does that play well with multiprocessing if the exc/tb is generated in the child and then formatted later in the parent?

Reply:
I can't find a good way to type annotate traceback objects, and `type(tb)` gives me a type class that I can't use. Going to leave these as pre-formatted `str` for now, and we can revisit if you know the right way.

Reply:
I'm good with this for now, but have you tried https://docs.python.org/3/library/types.html#types.TracebackType ?